ext/webidl/00_webidl.js

```diff
@@ -682,11 +682,19 @@
       }
     }
 
-    return function (V, opts = {}) {
+    return new Function(
+      "makeException",
+      "type",
+      "defaultValues",
+      "allMembers",
+      `return function (V, opts = {}) {
```

Comment:
This will cause the process to fail if `--disallow-code-generation-from-strings` flag is used (https://github.com/denoland/deno/issues/16535)